game-core/src/main/java/games/strategy/engine/player/IRemotePlayer.java

```diff
@@ -1,15 +1,259 @@
 package games.strategy.engine.player;
```

Comment:
I see IGamePlayer and IRemotePlayer are merged.

There is an attempt to make an abstraction in the code base to hide when elements are actually across the network such that the network layer becomes transparent.

Are all players going to be `IRemotePlayer` instances? Have you considered perhaps renaming `IRemotePlayer` to `IGamePlayer`, would that make sens at all?

Reply:
I'm open to renaming stuff.
I'd have actually renamed the interface to just `IPlayer`/`Player` because there is only a single player left.
Basically all classes that implemented `IGamePlayer` implemented `ITripleAPlayer` as well, both of those interfaces extended `IRemotePlayer`, which is why I could merge those 3 interfaces no problem

Reply:
Perhaps `GamePlayer` instead of `IRemotPlayer`?

Reply:
Would you be unhappy with just `Player`? 
I don't think that name is taken yet

Reply:
I'm not used to that as a name yet, still used to `GamePlayer`. It feels more specific, as opposed to say a player in lobby or a player in a network game staging screen. It might be my habituation though.

Ultimately no, with the above caveat - `Player` seems fine

Reply:
IMO just `Player` is consistent with `PlayerId` or `TripleAPlayer`, so that's what I'll be going with.